routes/users.js

```diff
@@ -130,8 +130,13 @@ router.post('/follow/:id', isAuth, async (req, res, next) => {
 //Editing should have name, contact, description, address ,location only
 router.post('/edit-profile', isAuth, async (req, res, next) => {
   try {
-
-
+         const user =  await User.find({_id:req.id})
+         user = {...user,...req.body.updates}
```

Comment:
@krish7777  will this work? the action is not yet fully written so I haven't tried it out. the updates will have the updated value.